### PR TITLE
clear dualportsupport code

### PR DESCRIPTION
**Pull request by lucas7788**

### CODE REVIEW

File: p2pserver/message/utils/msg_handler.go [hunk 1]
```diff
@@ -245,227 +244,141 @@ func VersionHandle(data *msgTypes.MsgPayload, p2p p2p.P2P, pid *evtActor.PID, ar
...
+	}
 
-		var msg msgTypes.Message
-		if s == msgCommon.INIT {
-			remotePeer.SetConsState(msgCommon.HAND_SHAKE)
-			msg = msgpack.NewVersion(p2p, true, ledger.DefLedger.GetCurrentBlockHeight())
-		} else if s == msgCommon.HAND {
-			remotePeer.SetConsState(msgCommon.HAND_SHAKED)
-			msg = msgpack.NewVerAck(true)
+	s := remotePeer.GetSyncState()
```
Comment (laizy) [inline p2pserver/message/utils/msg_handler.go]: rename to `GetState`, and remove `Sync` from other function name

File: p2pserver/peer/peer.go [hunk 2]
```diff
@@ -317,13 +317,13 @@ func (this *Peer) SetHttpInfoPort(port uint16) {
 func (this *Peer) UpdateInfo(t time.Time, version uint32, services uint64,
 	syncPort uint16, nonce uint64, relay uint8, height uint64, softVer string) {
 
-	this.SyncLink.UpdateRXTime(t)
+	this.RecvLink.UpdateRXTime(t)
 	this.base.SetID(nonce)
 	this.base.SetVersion(version)
 	this.base.SetServices(services)
 	this.base.SetSyncPort(syncPort)
```
Comment (laizy) [inline p2pserver/peer/peer.go]: rename to `SetPort`

File: p2pserver/peer/peer.go [hunk 3]
```diff
@@ -129,8 +129,8 @@ func (this *PeerCom) GetSoftVersion() string {
 type Peer struct {
 	base      PeerCom
 	cap       [32]byte
-	SyncLink  *conn.Link
-	syncState uint32
+	RecvLink  *conn.Link
```
Comment (laizy) [inline p2pserver/peer/peer.go]: why name it `Recv`?

File: p2pserver/message/types/version.go [hunk 4]
```diff
@@ -31,13 +31,14 @@ type VersionPayload struct {
...
 	HttpInfoPort uint16
-	ConsPort     uint16
-	Cap          [32]byte
-	Nonce        uint64
-	StartHeight  uint64
-	Relay        uint8
-	IsConsensus  bool
-	SoftVersion  string
+	//TODO
+	ConsPort    uint16
```
Comment (Honglei-Cong) [inline p2pserver/message/types/version.go:35]: remove ConsPort and IsConsensus ?

File: p2pserver/peer/peer.go [hunk 5]
```diff
@@ -335,26 +268,24 @@ func (this *Peer) GetSoftVersion() string {
 
 //AttachSyncChan set msg chan to sync link
 func (this *Peer) AttachSyncChan(msgchan chan *types.MsgPayload) {
-	this.SyncLink.SetChan(msgchan)
+	this.Link.SetChan(msgchan)
 }
 
 //AttachConsChan set msg chan to consensus link
 func (this *Peer) AttachConsChan(msgchan chan *types.MsgPayload) {
```
Comment (Honglei-Cong) [inline p2pserver/peer/peer.go]: remove this function?

File: p2pserver/peer/peer.go [hunk 6]
```diff
@@ -198,86 +181,36 @@ func (this *Peer) SetHeight(height uint64) {
...
-}
-
-//SetConsPort set peer`s consensus port
-func (this *Peer) SetConsPort(port uint16) {
-	this.ConsLink.SetPort(port)
+	return this.Link.GetPort()
 }
 
 //SendToSync call sync link to send buffer
 func (this *Peer) SendToSync(msgType string, msgPayload []byte) error {
```
Comment (Honglei-Cong) [inline p2pserver/peer/peer.go]: remove this function.

Move all its code to Peer.SendRaw() ?

File: p2pserver/message/types/version.go (hunk 4 above)
Comment (laizy) [reply p2pserver/message/types/version.go:35]: change packet format is incompatible, I suggest adding `TODO` document to remind us to remove those when we have the chance to upgrade the network layer.

File: p2pserver/peer/peer.go (hunk 6 above)
Comment (lucas7788) [reply p2pserver/peer/peer.go]: just rename SendToSync to SendTo

**Review by Honglei-Cong — APPROVED** [review]

**Pull request merged**